common-tools/clas-detector/src/main/java/org/jlab/detector/decode/DetectorEventDecoder.java

```diff
@@ -45,6 +45,18 @@ public DetectorEventDecoder(boolean development){
         }
     }
 
+    public void setTimestamp(String timestamp) {
+        translationManager.setTimeStamp(timestamp);
+        fitterManager.setTimeStamp(timestamp);
+        scalerManager.setTimeStamp(timestamp);
+    }
+
+    private void setVariation(String variation) {
```

Comment:
Since this method is added, should there be a command line option to choose the variation?